api/nakadi-event-bus-api.yaml

```diff
@@ -925,6 +1173,67 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the
+      number of clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'consumer_group'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        format: uuid
+        readOnly: true
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+      owning_application:
+        type: string
+        example: 'gizig'
+        description: |
+          The id of application owning the subscription. The reading and modification of subscription will be limited
+          to this application id.
```

Comment:
It is related to the discussion above about rewinds and having opened multiple streams at the same time. If one allows to open multiple stream on the same set of events in an application, it is to restrictive to call the parameter `owning_application` because this would lead to the same stream. In this case a more general term as `client/consumer_group_id` is in my opinion more appropriate, as it straight forward suggests that you can use this identifier to create a group of clients/consumers that read from same event stream.